plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/product-details/style.scss

```diff
@@ -36,3 +37,76 @@
 		}
 	}
 }
+
+/*
+ * Minimal style.
+ */
+.editor-styles-wrapper .wp-block-woocommerce-product-details.is-style-minimal,
+body:not(.wp-admin) .wp-block-woocommerce-product-details.is-style-minimal .woocommerce-tabs {
```

Comment:
I'm a bit curious about why we need two different selectors for this. Is that for specificity reasons so the minimal styles override the defaults? If so, I would add a comment mentioning that. Also, I would always try to have the same selector in the frontend and the editor if possible.

Reply:
Yes. Thanks for flagging this! 

After reviewing the CSS and considering your suggestion to use a new `editor.scss` file, I believe we can greatly improve the organization of our rules. I went through it again and made some changes: https://github.com/woocommerce/woocommerce/pull/42965/commits/0ef7882b9966cad9ad1a1a2c207a591fddde6a84